.travis.yml

```diff
@@ -1,57 +0,0 @@
-language: php
-
-cache:
-  directories:
-    - $HOME/.composer/cache
-    - vendor
-
-matrix:
-  include:
-    - php: 8.0
-      env: SYMFONY_VERSION=6.0.*
-    - php: 8.0
-      env: SYMFONY_VERSION=5.0.*
-    - php: 8.0
-      env: SYMFONY_VERSION=4.4.*
-    - php: 7.4
-      env: SYMFONY_VERSION=5.0.*
-    - php: 7.4
-      env: SYMFONY_VERSION=4.4.*
-    - php: 7.4
-      env: SYMFONY_VERSION=4.3.*
-    - php: 7.3
-      env: SYMFONY_VERSION=5.0.*
-    - php: 7.3
-      env: SYMFONY_VERSION=4.4.*
-    - php: 7.3
-      env: SYMFONY_VERSION=4.3.*
-    - php: 7.2
-      env: SYMFONY_VERSION=4.2.*
-    - php: 7.1
-      env: SYMFONY_VERSION=3.4.*
-    - php: 7.1
-      env: SYMFONY_VERSION=2.8.*
```

Comment:
this can not have worked anymore, because the composer.json requires symfony components in version 3.4 or newer...

i will drop symfony 2 support in the test matrix.